### DISCUSSION TITLE
FIBO instance Data?

### DISCUSSION THREAD

**Discussion by bblfish** [edmcouncil/fibo#1368] [General]
Hi,

   I am new to fibo, though quite knowledgeable in OWL and Semweb technologies. 
As I understand Fibo is an ontology. Is there any good quality public fibo data available. Perhaps from a company that went bankrupt or perhaps fake but believable data? Real data about some famous event would be more interesting of course.

**Comment by dallemang** [comment]
We have some data we use in our training courses, available at data.world (
https://data.world/edm-council)  It only pertains to swaps.  It actually is
somewhat believable; the schema was taken from one shown to us by a
government regulator, and the data, while made up, has enough connectivity
that you can find interesting stuff about it.

 We also have some more up-to-date data in the FIBO git repository, that we
need to incorporate into the data.world projects (which I haven't done
yet).

There's also validation data in
https://github.com/edmcouncil/fibo/tree/master/etc/testing/data/fnd

I hope that gets you started.

Dean

**Comment by ElisaKendall** [comment]
Hi Henry,

If you are working from the FIBO github as opposed to downloading FIBO from the council web site, there is a load file, called AboutFIBOProd.rdf at the top level under fibo.  That file and the related catalog in the fibo folder will load all of the production ontologies and a number of examples all at once in Protege.  The catalog is relative, so you should be able to use that directly.  

From there, there are three kinds of individuals in FIBO: (1) nominals, (2) reference data (currency codes, exchange codes and related business centers, and FpML interest rates), and (3) examples.  Many of our examples are based on real-world organizations and instruments, though some of the data about those organizations or instruments may be dated.

I gave a talk that walked through some of the examples for the recent KCG conference, which I'll add here.  Hopefully you'll be able to find them and poke around.  

Best,

Elisa

[FIBOCooksTour-KGC2020-v4.pdf](https://github.com/edmcouncil/fibo/files/4754688/FIBOCooksTour-KGC2020-v4.pdf)

**Comment by bblfish** [comment]
Thanks @ElisaKendall. Using the github repo version of FIBO works much better, somehow. (I was only able to get Protege to load yesterday using the FIBO URL, but as it seemed to download each of the files one after the other it took a lot of time). I have found those example entities. 

I still can't quite tell if the reasoner has finished reasoning on the data, though. Is there a reasoner that is recommended for FIBO in Protege?

**Comment by bblfish** [comment]
Thanks for the slides @ElisaKendall.  I see you mention LEIs there a few times. You may be interested in a Linked Data use case that I have where FIDO could also be used which I call WoN. Here is a very short overview of it https://medium.com/@bblfish/use-cases-for-the-web-of-nations-361c24d5eaee

**Comment by ElisaKendall** [comment]
Hi Henry, 

Thank you for the link - I'll take a look.  With respect to reasoners, HermiT doesn't complete on my laptop - it might on a server, but I typically run out of memory relatively quickly.  For testing any of the production ontologies I use an old version of Pellet.  It's available on github still.  It does complete on all of FIBO Prod after a few hours, as long as you have sufficient memory.  In github you'll find there are various 'load files', under some of the modules in FIBO, so that you don't have to load the whole thing.  You'll find them in FND, BE, FBC, SEC, and IND, and a partial one in DER, fyi.  The smallest load files are named 'All<x>', where x is the module name.  If they include either reference data or examples they take a lot longer.

Hope that's helpful,

Elisa

**Comment by rivettp** [comment]
If you're interested in LEIs, Henry, there is a complete RDF dataset publicly available here https://data.world/gleif/lei-data (free signup needed), which is something I worked on funded by GLEIF: it has not been publicly announced yet. The ontology is not quite the same as FIBO (you can see them on Linked Open Vocabularies https://lov.linkeddata.es/dataset/lov/terms?q=gleif) but was influenced by and influenced FIBO and cut-down to be fit for a more limited purpose. The example queries on data.world https://data.world/gleif/getting-started-with-lei-rdf show linking with the OMG "Languages, Countries and Codes" dataset https://data.world/rivettp/omg-lcc which has RDF representations of all the ISO countries and subdivisions.
Regarding your Medium article there is new work being discussed that you may be interested in: if so please drop me an email to pete.rivett at federatedknowledge.com and I'll put you in touch with the right person.
